### PR TITLE
Add getCustomer in Transaction

### PR DESCRIPTION
**Pull request by dara123**

### CODE REVIEW

**Comment by dara123** [comment]
devs please review

File: src/Entities/Transaction.php [hunk 1]
```diff
@@ -330,4 +330,16 @@ public function getRedirectUrls()
     {
         return $this->getAttribute('redirectUrls');
     }
+
+    /**
+     * @return null|Customer
+     */
+    public function getCustomer()
+    {
+        if ($this->hasEmbeddedResource('customer')) {
```
[the remark this answers is not in the document]
Comment (2e3s) [reply src/Entities/Transaction.php]: How can someone set it?

[the remark this answers is not in the document]
Comment (dara123) [reply src/Entities/Transaction.php]: that's why we need to check make we have customer in the response

[the remark this answers is not in the document]
Comment (dara123) [reply src/Entities/Transaction.php]: `$client->transactions()->search(['expand' => 'customer']);`

[the remark this answers is not in the document]
Comment (dara123) [reply src/Entities/Transaction.php]: `$params = new ParamBag();
$params->expand(['customer']);
$client->transactions()->search($params);`

[the remark this answers is not in the document]
Comment (slavcodev) [reply src/Entities/Transaction.php]: > How can someone set it in PHP SDK?

```
$transaction = $client->transactions()->load('foo', ['expand' => 'customer']);
```

**Pull request merged**